docs/self-managed/components/orchestration-cluster/core-settings/overview.md

```diff
@@ -0,0 +1,39 @@
+---
+id: overview
+title: "Overview"
+description: "The central documentation hub for shared configuration and conceptual topics across the Camunda 8 Orchestration Cluster."
+---
+
+Welcome to the central documentation hub for shared configuration and conceptual topics across the Camunda 8 Orchestration Cluster components: Operate, Tasklist, and Identity.
+
+## Purpose
+
+With the release of version 8.8 and beyond, configuration and operational concepts for the orchestration cluster have been unified. This means:
+
+- Many configuration options previously documented separately for each component are now consolidated into a single, centralized configuration file and management approach.
+- Several conceptual topics—such as security headers, authentication, data retention, and usage metrics—apply across all orchestration cluster components rather than to just one.
+- This shared section provides a single source of truth for these cross-cutting concerns, making it easier to find, understand, and maintain orchestration cluster configuration and concepts.
+
+## What you will find here
+
+This section includes:
+
+- **Unified configuration topics:** Centralized documentation on all cluster-wide configuration settings grouped by theme (e.g., security, logging, monitoring).
+- **Conceptual documentation:** Explanations of cluster-wide behaviors, features, and best practices relevant across multiple components.
+- **Migration and legacy information:** Guidance related to upgrading and migrating to version 8.8+, including notes on deprecated features and legacy support.
+- **Cross-component topics:** Information on shared aspects such as HTTP security headers, authentication mechanisms, data retention policies, and usage metrics.
+
+## Why this matters
+
+Previously, configuration and concepts were scattered across multiple component-specific docs (Operate, Tasklist, Identity), which could cause confusion and duplication. Centralizing this content:
```

Comment:
I wonder if we should put so much emphasis on the before-vs-after comparison here. For anyone who starts with 8.8 or later, knowing this doesn't matter. Would this explanation better fit into the migration guide instead (and here we only explain the as-is strucutre so that people know where they can find which config)?